etcdctl/README.md

```diff
@@ -1505,6 +1505,26 @@ CHECK provides commands for checking properties of the etcd cluster.
 
 CHECK PERF checks the performance of the etcd cluster for 60 seconds. Running the `check perf` often can create a large keyspace history which can be auto compacted and defragmented using the `--auto-compact` and `--auto-defrag` options as described below.
 
+Notice that different workload models use different configurations in terms of number of clients and  throughtput. Here is the configuration for each load:
```

Comment:
nit: there are two spaces between `and` and `throughput` :)

Reply:
Hello @ahrtr Thanks for your feedback. I fixed the double space. BTW, I found another one later in the file.